tsconfig.json

```diff
@@ -19,7 +19,8 @@
     "strict": true,
     "skipLibCheck": false,
     "noUnusedLocals": true,
-    "noUnusedParameters": true
+    "noUnusedParameters": true,
+    "strictPropertyInitialization": false
```

Comment:
You do not need to set this to false. Use **undefined** or **definite assignment assertion** for initialization

Reply:
In or code not needed, though mock file will throw many errors.